lib/yrewrite_url_schemes.php

```diff
@@ -46,7 +46,7 @@ public function getCustomUrl(rex_article $art, rex_yrewrite_domain $domain)
         }
 
         if ($domain->getStartId() == $art->getId()) {
-            if ($domain->getStartClang() == $art->getClang()) {
+            if (!$domain->isStartClangAuto() && $domain->getStartClang() == $art->getClang()) {
```

Comment:
```suggestion
            if ((!method_exists($domain, 'isStartClangAuto') || !$domain->isStartClangAuto()) && $domain->getStartClang() == $art->getClang()) {
```

So könnte man kompatibel bleiben zu älteren yrewrite-versionen. (ungetestet)